tomviz/modules/ModuleSlice.cxx

```diff
@@ -365,6 +368,12 @@ void ModuleSlice::dataUpdated()
   emit renderNeeded();
 }
 
+void ModuleSlice::pipelineFinished()
+{
+  m_producer->SetOutput(dataSource()->producer()->GetOutputDataObject(0));
```

Comment:
Let's add a FIXME comment here to state that the pipeline connection is broken, and we should investigate a more elegant fix after the release.

Reply:
How does this account for multiple channel too - it is always selecting channel 0 isn't it?